src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -618,7 +618,7 @@ private void ReceiveSingleDocumentsBatch(DocumentsOperationContext documentsCont
                         }
                         else
                         {
-                            _log.Info("Failed to receive documents replication batch. This is not supposed to happen, and is likely a bug.", e);
+                            _log.Info("Failed to receive documents replication batch.", e);
```

Comment:
if this is expected maybe we shouldn't log an error here, but simply log the connection is shutting down?

Reply:
Ping

Reply:
I believe we should keep logging the exception because it’s important to capture the details of the exception. I considered adding a condition like `else if (e is ObjectDisposedException)`, but I'm not confident we'll always encounter `ObjectDisposedException`. by "expected" I mean that while exceptions might occur under certain conditions, they probably don’t indicate a critical failure.

Reply:
Can we detect that this is the case?
Maybe we have a cancellation token?

Reply:
@grisha-kotler the error is occurring on the outgoing side, which then triggers an error when receiving items on the incoming side. since this does not represent a graceful flow, I think it is important to keep logging the exception here to capture the details accurately.

Reply:
So let's have a dedicated log for this.
If we get object disposed, log something like:
`Shutting down the replication conncetion` ...

Reply:
done